pkg/jobrunaggregator/jobtableprimer/job_typer.go

```diff
@@ -15,6 +15,8 @@ func newJob(name string) *jobRowBuilder {
 	switch {
 	case strings.Contains(name, "gcp"):
 		platform = gcp
+	case strings.Contains(name, "arm64"):
```

Comment:
Are these actually platforms, not a new field type call architecture?

Reply:
In theory there are architecture, thats where my original confusion came along as we want it to be part of separate field